prow/apis/prowjobs/v1/types_test.go

```diff
@@ -23,9 +23,13 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
-	"github.com/google/gofuzz"
+	fuzz "github.com/google/gofuzz"
 )
 
+func pStr(str string) *string {
+	return &str
```

Comment:
Initially thought this could cause invalid memory address as `str` is passed in as a copy of the value, so the entire memory address in only valid within the scope of this function. then later on figured out how go is smart on this https://www.ardanlabs.com/blog/2017/05/language-mechanics-on-escape-analysis.html, it's good to learn